class UnexpectedDataFormatError(Exception):
    """Исключение, возникающее при неожиданном формате данных."""
    pass
